vars/provision.groovy

```diff
@@ -1,48 +1,53 @@
+
 def createResources(repoName, pr) {
-  if(manifest.azure) {
-    deletePrResources(repoName, pr)   
-    createAllResources(pr)
+  def filePath = 'provision.azure.yaml'
+  if(fileExists(filePath)) {
+    //deletePrResources(repoName, pr)   
+    createAllResources(filePath, repoName, pr)
   }
 }
 
-def readManifest() {
-  return manifest
+def fileExists(filePath){
+  return sh(returnStdout: true, script: "test -f $filePath && echo true || echo false")
```

Comment:
You might want a `.trim()` on the `sh` command as it will return a newline after the `true` or `false`.
There is a  built in Groovy function `fileExists` that has been used elsewhere, i.e. https://github.com/DEFRA/ffc-jenkins-pipeline-library/blob/0522eb953d4f794a6b16c004e9b269c5fb65d20d/vars/buildDotNetCore.groovy#L37 so you could remove this method altogether.

Reply:
@FayToward did try to tell me we'd done that somewhere before!

Thanks, updated.